app/models/resource.rb

```diff
@@ -0,0 +1,5 @@
+class Resource < VersionedRecord
+  belongs_to :type, class_name: "Resourcetype"
+
+  validates :title, presence: true
+end
```

Comment:
will add
* `validates :resourcetype_id, presence: true` (will rename field)

also we should likely add the following in https://github.com/dumparkltd/marine-defrag-server/tree/measure-resources (not sure if needed though
* `has_many :measures, through: :measure_resources, inverse_of: :resources`